app/controllers/staff/training_sessions_controller.rb

```diff
@@ -43,7 +44,10 @@ def update
     @current_training_session.update(changed_params)
 
     if params['dropped_users'].present?
-        @current_training_session.users -= User.where(username: params['dropped_users'])
+      @current_training_session.users -= User.where(username: params[ 'dropped_users'])
+      User.where(username: params[ 'dropped_users']).each do |user|
```

Comment:
Whoops, I wrote this wrong. It meant this would blow up if the user didn't have a certification. So you only need the `cert.present?` not the `user.present?`